lib/netsuite/records/custom_field_list.rb

```diff
@@ -113,12 +113,15 @@ def extract_custom_field(custom_field_data)
             if type == "platformCore:SelectCustomFieldRef"
               attrs[:value] = CustomRecordRef.new(custom_field_data[:value])
             elsif type == 'platformCore:MultiSelectCustomFieldRef'
-              attrs[:value] = custom_field_data[:value].map do |entry|
-                CustomRecordRef.new(entry)
+              # if only one value of multiselect is selected it will be a hash, not an array
+              if attrs[:value].is_a?(Array)
+                attrs[:value] = custom_field_data[:value].map { |entry| CustomRecordRef.new(entry) }
+              else
+                attrs[:value] = CustomRecordRef.new(custom_field_data[:value])
```

Comment:
This was recently fixed: https://github.com/NetSweet/netsuite/commit/3377c971d0cb727d81f4b4bc6e30edfbdfaccfd1

Can you remove this change from your PR?